src/components/tabs/js/tabsController.js

```diff
@@ -673,7 +673,7 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
       containerWidth = elements.canvas.clientWidth,
       xsBreakpoint = 600,
 
-      // See https://material.io/design/components/tabs.html#spec which has been updated to 90px.
+      // See https://material.io/archive/guidelines/components/tabs.html which has been updated to 90px.
```

Comment:
Let's change this to
`// See https://material.io/archive/guidelines/components/tabs.html#tabs-specs`
and remove the other part about `90px` since that refers to the 2018 spec.

Reply:
This has been updated :chicken: